postcommit.py

```diff
@@ -0,0 +1,642 @@
+		for x in rev_list:	 					# Iterate until branch base point
+			rev_cmd = "git branch --contains " + x
+			rev_cmd = status_check(rev_cmd,self.module_dir)
+			rev_cmd = str(rev_cmd[0]).split('\n')
+			if(len(rev_cmd) > 2): 
+				validate = True
+				break
+			else:
+				changeset.append(x)
+		return validate
+
+	
+#       ***************************************************************************************************************
+#	branchlog :
+#		
+#			Find a number of commits between tow changsets
+#
+#			@param : branchname
+#			@param : oldId
+#			@param : newId
+#			
+#			@return : None        						
+#	*************************************************************************************************************** 
+    	def branchlog(self, branchname=None, oldId=None,newId=None):
+        	pretty_format = " --pretty='format:CMHash:%H %h##CMParent:%P##%p##CMAuthor:%an %ae##CMDate:%ci##CMMessage:%n%s%n%bLOGENDS'"
+      	 	if oldId == "0000000000000000000000000000000000000000":
+			changeset = []
+			max_count = 50
+			commit_count = 0
+			skip = 0
+			cmd = "git rev-list --count "+branchname
+			commit_count = status_check(cmd,self.module_dir)
+			commit_count = int(commit_count[0])
+			if(commit_count < 50):
+				max_count = commit_count
+			while commit_count>=max_count:
+				validate = self.findNewBranchCommits(changeset,max_count,skip,newId,branchname)
+				if(validate):
+					break
+				max_count+=50
+				skip+=50
+			length = len(changeset)	
+			if(length == 1):
+           			cmd = "git log --numstat -1 " +  newId + pretty_format
+			else:
+				cmd = "git cat-file -p "+ changeset[length-1]
+				parent_rev = status_check(cmd,self.module_dir)
+				parent_rev = str(parent_rev[0]).split('\n')[1]
+				parent_rev_pattern = re.compile('([a-z0-9]+)')
+				result = parent_rev_pattern.findall(str(parent_rev))
+				if(result[0] != "parent"):
+					cmd = "git log --numstat " + changeset[0] + pretty_format
+				else:
+					cmd = "git log --numstat " +  changeset[length-1] + '~1..' + changeset[0] + pretty_format
+		elif newId == "0000000000000000000000000000000000000000":
+			cmd = ""			# For a branch deletion command statement is empty
+		else:
+			cmd = "git log --numstat " + oldId + '..' +  newId + pretty_format
+        	commits = status_check(cmd, self.module_dir)
+        	self.parseCommits(str(commits[0]), branchname)
+
+	
+#	***************************************************************************************************************
+#	updateMergeDetails :
+#
+#			This method used to find how many files added,removed,modified in single changeset.Also it 
+#			shows number of added and deleted lines in decimal notation for each files. 
+#
+#			@param : log
+#			@param : parents
+#			@param : parent_short
+#			
+#			@return : log         	
+#					
+#	*************************************************************************************************************** 
+    	def updateMergeDetails(self, log, parents, parents_short):
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename method "updateMergeDetails" to match the regular expression ^[a-z_][a-z0-9_]{2,}$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=python%3AS100)